### ISSUE TITLE
Add Client methods

### ISSUE THREAD

**Issue by TomAugspurger** [mapd/pymapd#34]
- [x] load (#38)
- [x] load_binary
- [x] get_tables (#37)
- [x] get_table_details (#37)

**Comment by TomAugspurger** [comment]
Looks like `load_table_binary` may be something like:

```python
binary_rows = []

for i in range(100):
    row = TRow([
        TDatum(TDatumVal(int_val=i)),
        TDatum(TDatumVal(real_val=float(i))),
        TDatum(TDatumVal(str_val=str(i)))
    ])
    binary_rows.append(row)
c.load_table_binary(s, 'baz', binary_rows)
```

**Comment by TomAugspurger** [comment]
This was closed by #53
